modules/nf-core/mmseqs/createdb/main.nf

```diff
@@ -0,0 +1,57 @@
+process MMSEQS_CREATEDB {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::mmseqs2=14.7e284"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mmseqs2:14.7e284--pl5321hf1761c0_0':
+        'biocontainers/mmseqs2:14.7e284--pl5321hf1761c0_0' }"
+
+    input:
+    tuple val(meta), path(sequence)
+
+    output:
+    tuple val(meta), path("${prefix}/") , emit: db
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    mkdir -p ${prefix}
+
+    mmseqs \\
+        createdb \\
+        ${sequence} \\
+        ${prefix}/seq \\
```

Comment:
Would it make sense to call it:
```suggestion
        ${prefix}/${prefix} \\
```
?
If yes, also modify the other instances in `output` and `stub` sections

Reply:
Yeah I was thinking about doing it like that as well! I added the seq because otherwise, MMSeqs creates hidden files in the directory like `.dbtype`and `.index` and was thinking that it will introduce errors downstream somewhere